### ISSUE TITLE
Use case: can this be used to output elisp

### ISSUE THREAD

**Issue by haikyuu** [Robert-van-Engelen/lua-to-lisp#1]
One interesting use case is to transpile neovim plugins to elisp for them to run in emacs.
Neovim uses 5.2 but that wouldn't be a big issue right?

This would be very nice to share code from the neovim community.

Also there is a vimscript to lua transpiler we can leverage to go the extra mile and port some things from Vim.

Nevertheless, this will require some other layer to allow plugin authors to  target both emacs and neovim using lua. But it's a very promising future

**Comment by Robert-van-Engelen** [comment]
@haikyuu yes, I think that would be possible, assuming that Lua's C API isn't used and there is no user data manipulation, because that's not trivial to do in Lisp (or needs a specific Lisp to translate to, to do the same). My transpiler implementation may still need a bit of work, because Lua multi-way assignments require Lisp code to perform multi-way assignments that I didn't finalize (got distracted). Like `x,y=y,x` and `x,y=f(a)` etc. is translated to Lisp, but it's not complete yet.

Anyway, it is worth a try to run the transpiler and see what you get. Having a practical use case helps to test and finish this work so everything works as we know is possible by the translation rules that are already carefully checked.
